vmngclient/api/templates/device_template/device_template.py

```diff
@@ -73,6 +75,10 @@ def parse_templates(cls, value):
                 output.append(template)
         return output
 
+    @validator("device_type")
```

Comment:
Enums are automatically converted to string by pydantic, no validator needed.
It might be suitable to define `DeviceModel` as:
```python
class DeviceModel(str, Enum)
    ...
```
Also I consider changing the type of field in validator (assigning value of different type than modelled) a bad practice. Please note that validators are executed both when constructed by library user and when parsing serialized data.

Reply:
Thanks, fixed

Reply:
ok looks like str is needed in .dict (not only model dump)